common/src/main/java/com/microsoft/identity/common/internal/net/HttpRequest.java

```diff
@@ -99,59 +63,6 @@ public Boolean apply(Exception e) {
     public static final String REQUEST_METHOD_OPTIONS = "OPTIONS";
     public static final String REQUEST_METHOD_PATCH = "PATCH";
 
-    private static final Set<String> HTTP_METHODS = new LinkedHashSet<>();
-
-    static {
-        HTTP_METHODS.add(REQUEST_METHOD_GET);
-        HTTP_METHODS.add(REQUEST_METHOD_POST);
-        HTTP_METHODS.add(REQUEST_METHOD_HEAD);
-        HTTP_METHODS.add(REQUEST_METHOD_PUT);
-        HTTP_METHODS.add(REQUEST_METHOD_DELETE);
-        HTTP_METHODS.add(REQUEST_METHOD_TRACE);
-        HTTP_METHODS.add(REQUEST_METHOD_OPTIONS);
-        HTTP_METHODS.add(REQUEST_METHOD_PATCH);
-    }
-
-    // class variables
-    private final URL mRequestUrl;
-    private final byte[] mRequestContent;
-
-    URL getRequestUrl() {
-        return mRequestUrl;
-    }
-
-    byte[] getRequestContent() {
-        return mRequestContent;
-    }
-
-    String getRequestContentType() {
-        return mRequestContentType;
-    }
-
-    String getRequestMethod() {
-        return mRequestMethod;
-    }
-
-    Map<String, String> getmRequestHeaders() {
```

Comment:
I see we've made an effort to be backcompat here -- I see this method was renamed to `getRequestHeaders()` in the superclass so in order to truly maintain backcompat we should preserve `getmRequestHeaders()` here and just call super (since the naming is fixed in the superclass)

Reply:
It looks like this method is package-private, so it shouldn't break MSAL/ADAL/Broker.

Reply:
ACK